src/shared/CMakeLists.txt.inc

```diff
@@ -12,10 +12,11 @@ include_directories(${shared_dir}/vartypes/enhanced)
 include_directories(${shared_dir}/vartypes/gui)
 include_directories(${shared_dir}/vartypes/xml)
 
-include_directories("/usr/include/eigen2")
+# include_directories("/usr/include/eigen2")  # update for eigen3 at toplevel 01/16
```

Comment:
I see. Okay, given that the primary platform (Ubuntu LTS) has Eigen3, then we could just remove this commented out line entirely.